src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  cursor: pointer;
+}
+
+.header {
+  display: flex;
+  margin: 0 50px;
```

```suggestion
  padding: 0 50px;
```
it can fix two failed tests